### PR TITLE
Show different backgrounds based on weather condition

### PR DESCRIPTION
**Pull request by BarokDG**
Issue #4

### CODE REVIEW

File: src/lib/getBackgroundImageFromWeatherCondition.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  Ash: {
+    imageSrc: null,
+  },
+  Squall: {
+    imageSrc: null,
+  },
+  Tornado: {
+    imageSrc: null,
+  },
+};
```
Comment (Reggie-Amedee-Tech) [inline src/lib/getBackgroundImageFromWeatherCondition.js:53]: Is there a way for us to programmatically fill in the keys/value pairs for this object? maybe a map function and a regex expression for naming the key?

File: src/lib/getBackgroundImageFromWeatherCondition.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import CloudsIMG from "../assets/images/clouds.jpg";
+import ClearIMG from "../assets/images/clear.jpg";
+import ThunderstormIMG from "../assets/images/thunderstorm.jpg";
+import RainIMG from "../assets/images/rain.jpg";
+import SnowIMG from "../assets/images/snow.jpg";
+
+const mapWeatherConditionToBackground = {
```
Comment (Reggie-Amedee-Tech) [inline src/lib/getBackgroundImageFromWeatherCondition.js:17]: Is there a way for us to programmatically fill in the keys/value pairs for this object? maybe a map function and a regex expression for naming the key?

Comment (BarokDG) [reply src/lib/getBackgroundImageFromWeatherCondition.js:17]: Not sure about this. Importing images is a very specific process with create-react-app. I will look into it. Thanks!

**Comment by utk09** [comment]
Please resolve the merge conflicts @BarokDG

File: src/lib/getBackgroundImageFromWeatherCondition.js (hunk 2 above)
Comment (BarokDG) [reply src/lib/getBackgroundImageFromWeatherCondition.js:17]: @Reggie-Amedee-Tech Found a solution. I can dynamically set the src for images if I put the images in the public folder. Have it setup locally. Will push once I'm done with an additional feature I'm working on.

**Pull request merged**